model.go

```diff
@@ -60,11 +60,15 @@ func (f *FlagModel) FormatPlaceHolder() string {
 	if f.PlaceHolder != "" {
 		return f.PlaceHolder
 	}
-	if f.Default != "" {
+	if len(f.Default) > 0 {
+		ellipsis := ""
+		if len(f.Default) > 1 {
+			ellipsis = "..."
+		}
 		if _, ok := f.Value.(*stringValue); ok {
-			return strconv.Quote(f.Default)
+			return strconv.Quote(f.Default[0]) + ellipsis
```

Comment:
I am not quite sure I understand the branching here if the value casts, and this is untested, I'd like some help...